### PR TITLE
Strong Motion Azimuth Deployment Guide/Doc

### PR DESCRIPTION
**Pull request by CBurton90**

### CODE REVIEW

**Review by mnaguit — COMMENTED** [review]
This is a good documentation, thanks @CBurton90 👍 A similar one for the structural arrays would surely be very handy for our data users.
The content is all good for me, I am just thinking if citing sensor examples for FBA/MEMS would be useful info (although not really relevant with azimuth)?

**Review by staylorofford — DISMISSED** [review]

**Comment by staylorofford** [comment]
Following on from what Muriel states, it may simply not be needed to say FBA/MEMS and just say it applies to all SM sensors outside of structural arrays

File: docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Regarding instrument installation, there are three types of strong motion site (excluding structural arrays):
+
+1. Sites in buildings
+2. Sites in GeoNet field infrastructure (cabinet, vault, VSAT hut)
+3. Sites in boreholes
+
+### 1) Sites in buildings
+
+Sensors at sites in buildings are installed inside cases which are typically aligned with the walls of the building. The orientation of these sensors should be in line with the edges of their host cases, which are aligned with the walls of the host building. In short, these sensors should be aligned with the walls of the host building. Their orientation can then be taken from the orientation of these walls. The miniSEED channel naming for these sites will be HNZ|1|2 (axial). If the building structure happens to be aligned to true north, there will be no change to this naming schema and the azimuth will be recorded as 0 (to maintain metadata consistency/continuity).
```
Comment (elidana) [inline docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md:15]: suggesting some minor wording modifications below:


Sensors at sites in buildings are generally aligned with the walls of the host building. These sensors installed inside cases which are typically aligned with the walls of the building. Their orientation should be in line with the edges of their host cases, which are aligned with the walls of the host building, and can then be taken from the orientation of these walls. The miniSEED channel naming for these sites will be `HNZ|1|2` (axial). If the building structure happens to be aligned to true north, there will be no change to this naming schema and the azimuth will be recorded as `0` (to maintain metadata consistency/continuity).

File: docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### 2) Sites in GeoNet field infrastructure (cabinet, vault, VSAT hut)
+
+Sensors at sites in GeoNet infrastructure are installed in light structures without cases. There is varying amounts of room for orienting sensors in these installations. Here it is desirable that sensors are true north-aligned, but if this is not possible or cannot be done accurately due to space constraints, an alignment with the host structure’s walls is acceptable. The orientation can then be taken from the orientation of these walls. For the majority that are true north-aligned the miniSEED channel naming will be HNZ|N|E (non-axial) and those in the rare instances that are aligned to the structure will be named HNZ|1|2 (axial).
```
Comment (elidana) [inline docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md:19]: apologies, that's me being ignorant: how can a user know the orientation of GeoNet field infrastructure walls? is that indicated somewhere in the metadata? Might be useful to specify how to get the "host structure's walls" perhaps?

File: docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+### 3) Sites in boreholes
+
+Sensors at sites in boreholes generally cannot be aligned to true north if the depth of the borehole is >10m as rotating the sensor is impratical at these depths. The orientation can then be taken from either an onsite method (surface sensor comparison) or offsite method (polarisation analysis), please see SEISMIC_SENSOR_ORIENTATION_POLARITIES.md for further information regarding these methods. The miniSEED channel naming for these sites will be HNZ|1|2 (axial). For shallow boreholes <10m rotating the sensor may be achievable and in these circumstances (if visbility allows an accurate measurement) the sensor will be oriented true north with miniSEED channel naming HNZ|N|E.
```
Comment (elidana) [inline docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md:23]: two things for this paragraph

1. can you please add a link to the `SEISMIC_SENSOR_ORIENTATION_POLARITIES.md`? 
    something like the thing below in the markdown should work (hopefully) 
    ```
    [SEISMIC_SENSOR_ORIENTATION_POLARITIES.md](SEISMIC_SENSOR_ORIENTATION_POLARITIES.md)
    ```
2. can you please add a short statement about were to find azimuth info?

**Review by elidana — COMMENTED** [review]
Hi @CBurton90 this one looks really great and incredibly useful!!

I have added some very minor comments, take it or leave it as you see fit!!

File: docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md (hunk 2 above)
Comment (salichon) [reply docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md:19]: The person of the field gets the true north orientation and check the horizontal con poenents within the building ?
Then informed into Delta azimuth informatrion ? 

The user doesn t need in general the buidling orientation  ~  it is assumed to be free field (no builiding interaction assumption) .

**Review by salichon — COMMENTED** [review]
Looks good to me

**Comment by salichon** [comment]
Kia Ora
 @CBurton90  It this good enough to be merged ?
 cheers :)

**Comment by CBurton90** [comment]
@salichon I just need to tidy up the question here then yes. Will put it on my list today, thanks for the reminder.

File: docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md (hunk 2 above)
Comment (CBurton90) [reply docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md:19]: The GeoNet infrastructure orientation is analogous to the strong motion sensor/recorder orientation (if true North alignment is not possible). We record that orientation/azimuth information in install/sensors.csv and install/recorders.csv. We don't need to tell the end user any specifics about the infrastructure orientation (as that can be inferred from the delta azimuth entries).

cc @elidana @salichon

**Comment by CBurton90** [comment]
https://github.com/GeoNet/delta/pull/1967#discussion_r1493965599

@salichon Is this a question? Yes for case 2 the field tech will measure true north outside of the enclosure/infrastructure and compare instrument alignment from that (to obtain the sensor/recorder alignment within the space constrained enclosure).

File: docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md (hunk 1 above)
Comment (CBurton90) [reply docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md]: I've updated : )

File: docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md (hunk 3 above)
Comment (CBurton90) [reply docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md]: Done, thanks!

For 2. that should be covered under "Find out more", lmk if something is missing!

**Comment by CBurton90** [comment]
> Following on from what Muriel states, it may simply not be needed to say FBA/MEMS and just say it applies to all SM sensors outside of structural arrays

@staylorofford @mnaguit I've removed any mention of FBA/MEMS. Thanks.

**Review by salichon — APPROVED** [review]

**Review by mnaguit — APPROVED** [review]

**Comment by CBurton90** [comment]
@mnaguit @salichon Could you please merge this one after a review : ) - I have no merge permissions!

**Pull request merged**